src/components/summary/_summary.scss

```diff
@@ -3,6 +3,7 @@ $summary-col-spacing: 1rem;
 $hub-row-spacing: 1.3rem;
 
 .summary {
+  @extend .u-mb-l;
```

Comment:
Sorry, I meant we should leave the summary bottom margin down to the user to add as they wish with a utility class, as we do with most elements.  They need to control spacing between components.

Reply:
Yeah I realised that and have fixed it